themes/basic/layouts/partials/head/scripts-header.html

```diff
@@ -1,10 +1,15 @@
 <!-- JS Libraries -->
+<script type="module">
+  import Alpine from '/js/alpinejs.js';
+  window.Alpine = Alpine;
+  Alpine.start();
+</script>
 
+{{ if .Site.Params.scrollScript }}
 <script type="module">
   import Scroll from '/js/toolkit-scroll.js';
   document.addEventListener('alpine:initializing', () => {
     Scroll(window.Alpine);
```

Comment:
Hm, wondering if there can be a race where this loads already after Alpine initialized, so would miss the setup... could check if there's window.Alpine object already, and if so, don't defer the init?